src/Bicep.Core/Analyzers/Interfaces/IBicepAnalyzerRule.cs

```diff
@@ -17,8 +17,6 @@ namespace Bicep.Core.Analyzers.Interfaces
     /// We are using a source generator that requires the fully qualified type name of this interface to not change.</remarks>
     public interface IBicepAnalyzerRule
     {
-        string AnalyzerName { get; }
```

Comment:
Interesting.  bicepconfig.json splits up the linter rules, with all current ones being under "core":

  "analyzers": {
    "core": {
      "verbose": false,
      "enabled": true,
      "rules": {
        "no-hardcoded-env-urls": {
        ...
        
 The assumption was that third-party rules would take different spots.  I guess that's why the warnings messages used to say "core".  Do we need to keep this distinction?

Reply:
Good point. I've updated these to:
`bicep core linter` - core linters
`bicep` - core diagnostics

Reply:
Perfect